benchmarks/ner_embeddings/scripts/set_attr.py

```diff
@@ -0,0 +1,54 @@
+import srsly
+import spacy
+
+from pathlib import Path
+from spacy.language import Language
+from spacy.pipeline import TrainablePipe
+from typing import Optional, Callable
+
+
+
```

Comment:
Have we run `black` for this file? I may have mistaken but I remember there are only `2` spaces after the import statements, not 3.